src/Sylius/Bundle/CoreBundle/Form/Type/ChannelType.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\Type;
+
+use Sylius\Bundle\ChannelBundle\Form\Type\ChannelType as BaseChannelType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Configurable channel form type.
+ *
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class ChannelType extends BaseChannelType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        parent::buildForm($builder, $options);
```

Comment:
We are overriding the ``sylius_channel`` type here, not creating a new type. So ``getParent`` does not work here. :)